ray_lightning/tests/test_ddp.py

```diff
@@ -227,6 +227,15 @@ def test_train_client(tmpdir, start_ray_client_server_2_cpus, num_workers):
     train_test(trainer, model)
 
 
+def test_test_multiple_dataloader_workers(tmpdir, ray_start_2_cpus, seed):
+    """Tests trainer.test with multiple workers for data loading."""
```

Comment:
I'm confused by this test. It seems like `BoringModel` uses the `test_dataloader`, which uses 1 worker. But, the test docstring says that multiple workers are used for data loading.

What am I missing?

Reply:
Ah sorry it should be >0 workers, not "multiple workers". Will update the comment.